tox.ini

```diff
@@ -13,3 +13,4 @@ changedir = .tox
 deps = coverage
 commands = coverage combine --rcfile {toxinidir}/.tox-coveragerc
            coverage report --rcfile {toxinidir}/.tox-coveragerc
+           coverage html --rcfile {toxinidir}/.tox-coveragerc -d {toxinidir}/htmlcov
```

Comment:
This is a little expensive to generate so it may be worth putting this in a separate tox env.  On the other hand, it's really nice that I can now rely on `htmlcov` hanging around.  It's totally OK if this stays but something to think about?